polling_stations/apps/file_uploads/templates/file_uploads/index.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block extra_page_css %}
+<style>
+  button:disabled {
+    background-color: lightgrey;
+    border-color: darkgrey;
+  }
+</style>
+{% endblock extra_page_css %}
+
+{% block content %}
+<div class="card">
+
+    <noscript>
+      <div class="card error-summary">
+          <h2>Javascript disabled</h2>
+          <p>To upload files, please enable javascript.</p>
+      </div>
+    </noscript>
+
+    <h2>Upload Files</h2>
+
+    <form id="file_upload_form" method="POST">
+
+        <div class="card error-summary" id="error" hidden></div>
+
+        <p>
+          <label for="council_id">Council</label>
+          <select id="council_id" name="council_id">
+            {% for council in councils %}
+            <option value="{{ council.council_id }}">{{ council.name }}</option>
```

Comment:
Two stretch goals here, post MVP:

1. select2 for filtering / type ahead
2. selecting a default from a GET param, for deep linking from the tracker sheet / inbox / whereever

Reply:
I've done the second one of these in b754ee7

Now that I've done the other UI bits in this PR, I suspect that's actually going to end up being the main way we use this, so there might be a bit more fiddling to do here.

Reply:
Select2 now irrelevant as I've totally removed the generic upload page (with a drop-down) in 
a192baf